### PR TITLE
was-873 - New feature spec covering operation of the Partnership page

### PR DESCRIPTION
**Pull request by autotelik**
Associated page object and page object for the add/remove partner review page
Shared examples for creating full journies

### CODE REVIEW

File: spec/features/partnership_spec.rb [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+      #review_page_object.dump
+
+      expect(page).to have_text t("enrollment_review.partnership.type_name_helper_text")
+
+      expect(page).to have_xpath "//a[contains(@id,'reviewing-change-businessdetails')]"
+
+      expect(page).to have_xpath "//a[contains(@href,'/dsc/reviews/other/#{enrollment.token}')]"
+    end
+  end
+=begin
```
Comment (rickclare) [inline spec/features/partnership_spec.rb]: I think Tim C has integrated partnerships into the confirmation email now (PR merged), so can these specs be uncommented?

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,8 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
+vendor/overcommit_bundle
```
Comment (rickclare) [inline .gitignore]: Out of interest, is this file/folder just something to help work-around RubyMine and Rubocop issues?

**Comment by rickclare** [comment]
Review done.  If we can add back the "confirmation email specs", then branch is good to merge.

File: .gitignore (hunk 2 above)
Comment (autotelik) [inline .gitignore]: trying to run overcommit as per travis i accidentally copied the whole cmd line which  installs gems to this location .. so I thought worth adding to .gitignore incase anyone manages to be as stupid as me !

**Pull request merged**